### ISSUE TITLE
manpage: wrong arguments described in list api

### ISSUE THREAD

**Issue by PatrykMatyjasek** [pmem/issues#74]
Wrong type of argument head described in manpage. Bad order of arguments head and dest
In manpage:
```
int pmemobj_list_insert(PMEMobjpool *pop, size_t pe_offset,
            PMEMoid dest, PMEMoid *head, int before, PMEMoid oid);

PMEMoid pmemobj_list_insert_new(PMEMobjpool *pop, size_t pe_offset,
           PMEMoid dest, PMEMoid *head, int before, size_t size,
           unsigned int type_num, void (*constructor)(void *ptr, void *arg), void *arg);

int pmemobj_list_remove(PMEMobjpool *pop, size_t pe_offset,
            PMEMoid oid, PMEMoid *head, int free);

int pmemobj_list_move(PMEMobjpool *pop,
           size_t pe_old_offset, size_t pe_new_offset,
           PMEMoid *head_old, PMEMoid *head_new, int tail, PMEMoid oid);
```
Header:
```
int pmemobj_list_insert(PMEMobjpool *pop, size_t pe_offset, void *head,
	PMEMoid dest, int before, PMEMoid oid);

PMEMoid pmemobj_list_insert_new(PMEMobjpool *pop, size_t pe_offset, void *head,
	PMEMoid dest, int before, size_t size, unsigned int type_num,
	void (*constructor)(void *ptr, void *arg), void *arg);

int pmemobj_list_remove(PMEMobjpool *pop, size_t pe_offset, void *head,
	PMEMoid oid, int free);

int pmemobj_list_move(PMEMobjpool *pop, size_t pe_old_offset,
	void *head_old, size_t pe_new_offset, void *head_new,
	PMEMoid dest, int before, PMEMoid oid);
```

Found in: nvml 0.1+b16-189-g177561b

**Comment by ldorau** [comment]
Fixed by pmem/nvml#249. Please, verify.

**Comment by PatrykMatyjasek** [comment]
Verified on 0.1+b16-208-g12c51c1
